core/app/finders/spree/taxons_finder.rb

```diff
@@ -0,0 +1,59 @@
+module Spree
+  class TaxonsFinder
+    def initialize(taxons, params)
+      @taxons = taxons
+      @params = params
+    end
+
+    def execute
```

Comment:
I thought we've agreed to use `call` for finders & services as opposed to `execute`

Reply:
👍